README.md

```diff
@@ -1,15 +1,21 @@
 # Source Engine Query Cacher
-![Downloads](
-https://img.shields.io/github/downloads/hyperxpro/SourceEngineQueryCacher/total)
 
-<p> High-Performance Source Engine Query Cacher for caching and responding A2S_INFO and A2S_PLAYER packets. </p>
+## This is a hard fork of
```

Comment:
This probably has to be reworked if you decide to approve the PR. I wasn't certain if you were interested so I made sure people get referred to you if the stumble across this on other platforms like GitLab